open_spiel/games/gin_rummy_test.cc

```diff
@@ -0,0 +1,559 @@
+// Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "open_spiel/games/gin_rummy.h"
+#include "open_spiel/games/gin_rummy/gin_rummy_utils.h"
+#include "open_spiel/spiel.h"
+#include "open_spiel/tests/basic_tests.h"
+
+namespace open_spiel {
+namespace gin_rummy {
+namespace {
+
+namespace testing = open_spiel::testing;
+
+bool ActionsContains(const std::vector<Action>& legal_actions, Action action) {
+  return std::find(legal_actions.begin(), legal_actions.end(), action) !=
+                   legal_actions.end();
+}
```

Comment:
Hah, that's the function I've been looking for. Template and move it into utils?

Reply:
Found an absl function that already does it! c_linear_search. Incorporated into new commit.